### ISSUE TITLE
One of examples at https://neptun.phys.au.dk/~bassler/TRiP is empty

### ISSUE THREAD

**Issue by grzanka** [pytrip/pytrip#40]
It looks that one of examples mentioned here https://neptun.phys.au.dk/~bassler/TRiP/ is empty.
Its description says 

> - TST003 : Water cube with box target

Inspected with hex editor (no duplicates are printed):
```
→ od -t x1 -An tests/TST003/tst003000.ctx
 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
*
```

Read with pytrip:
```
# grzanka at gajewski-dell in ~/PycharmProjects/pytrip on git:master ✖︎ [17:57:03]
→ cat tests/bug.py                       
from pytrip import CtxCube

c = CtxCube()
c.read("tests/TST003/tst003000.ctx")

print("shape", c.cube.shape)
print("element [0][0][0]", c.cube[0][0][0])
print("min value", c.cube.min())
print("max value", c.cube.max())

# grzanka at gajewski-dell in ~/PycharmProjects/pytrip on git:master ✖︎ [17:57:06]
→ PYTHONPATH=. python3 tests/bug.py
shape (300, 512, 512)
element [0][0][0] 0
min value 0
max value 0
```

**Comment by grzanka** [comment]
@nbassler can you take a look ?

**Comment by grzanka** [comment]
TST0002 (referred as `TST002 : Water cube with spherical target`) seems also to have empty CT cube:

```
# grzanka at gajewski-dell in ~/PycharmProjects/pytrip on git:master ✖︎ [18:21:50]
→ cat tests/bug.py                                            
import sys
from pytrip import CtxCube

c = CtxCube()
c.read(sys.argv[1])

print("shape", c.cube.shape)
print("element [0][0][0]", c.cube[0][0][0])
print("min value", c.cube.min())
print("max value", c.cube.max())

# grzanka at gajewski-dell in ~/PycharmProjects/pytrip on git:master ✖︎ [18:21:55]
→ PYTHONPATH=. python3 tests/bug.py tests/TST002/tst002000.ctx
shape (300, 512, 512)
element [0][0][0] 0
min value 0
max value 0

```

**Comment by nbassler** [comment]
Water has HU 0?

**Comment by grzanka** [comment]
Me stupid, you are right.

But on the other hand - does it make sense to have different .ctx and .hed files in examples TST001-TST003 ?

**Comment by nbassler** [comment]
That can be discussed. I think in that case we should separate the prototype CTs into a directory for itself. Would of course be nice to decrease the footprint.
An argument against it: it might increase confusion. Imagine user looking at TST002 dos cube, and then user cannot find related CT cube anywhere.

**Comment by grzanka** [comment]
And what about saving a .ctx.gz file (or .gz file which has .hed + .ctx inside) and adding to pytrip the capability of reading gzipped files ?

It shouldn't be that hard, and can be achieved using following items:
 - http://docs.scipy.org/doc/numpy/reference/generated/numpy.loadtxt.html (` If the filename extension is .gz or .bz2, the file is first decompressed. `)
 -  http://docs.scipy.org/doc/numpy/reference/generated/numpy.fromfile.html (takes file object as argument)
 - https://docs.python.org/3.5/library/gzip.html (returns a file object for gz file)

**Comment by grzanka** [comment]
OK, it will be a separate issue. Fixing it will also close this discussion.

**Comment by nbassler** [comment]
Closed this and #52 via #59
